src/index.ts

```diff
@@ -333,6 +341,38 @@ export async function install(
     };
     const packageBundle = await rollup.rollup(inputOptions);
     await packageBundle.write(outputOptions);
+
+    if (nomodule) {
+      function rollupResolutionHelper() {
+        return {
+          name: 'rename-import-plugin',
+          resolveId(source) {
+            // resolve from import map
+            if (importMap[source]) {
+              return importMap[source];
+            }
+            // resolve web_modules
+            if (source.includes('/web_modules/')) {
+              const suffix = source.split('/web_modules/')[1];
+              return {id: path.join(destLoc, suffix)};
+            }
+            // null means try to resolve as-is
+            return null;
+          },
+        };
+      }
+      try {
+        const noModuleBundle = await rollup.rollup({
+          input: nomodule,
+          plugins: [...inputOptions.plugins, rollupResolutionHelper()],
+        });
+        await noModuleBundle.write({file: path.resolve(destLoc, nomoduleOutput), format: 'iife'});
+        spinner.info(`${chalk.bold('snowpack')} created nomodule bundle: ${nomoduleOutput}`);
+      } catch (e) {
+        spinner.warn(`${chalk.bold('snowpack')} error building nomodule bundle: ${e}`);
```

Comment:
Followed the logic in your other comment. If it fails for whatever reason, it’ll err here (like for dynamic imports). However, any nomodule errors **won’t block snowpack from working**.

Reply:
+1, that sounds like the right tradeoff. Although I think `${e}` just translates to `[Error ...]`, so you may want to do `${e.message}`. Also while we're here, I'd slightly prefer `err` or even `error` over `e` :)

A test on the error behavior would be great, but not at all required.

Reply:
Changed `e` to `err` but can’t think of a test to write for the error behavior. It’s actually pretty darn resilient now that we added `inlineDynamicImports`. 🤔 I can’t think of a scenario where a nomodule bundle would easily fail, but the normal snowpack install wouldn’t. Even if you specify fake modules, Rollup will just treat them as external dependencies and throw a warning, but the `nomodule` bundle still outputs.

Reply:
👍 that works for me, if anyone encounters an error we can add a test then